cmd/azure.go

```diff
@@ -216,13 +212,14 @@ func (a AzureObjects) ListMultipartUploads(bucket, prefix, keyMarker, uploadIDMa
 		return result, nil
 	}
 	if len(resp.Parts) > 0 {
-		result.Uploads = []uploadMetadata{{prefix, prefix, time.Now(), ""}}
+		result.Uploads = []uploadMetadata{{prefix, prefix, time.Now().UTC(), ""}}
 	}
 	return result, nil
 }
 
 // NewMultipartUpload - Use Azure equivalent CreateBlockBlob.
 func (a AzureObjects) NewMultipartUpload(bucket, object string, metadata map[string]string) (uploadID string, err error) {
+	// Azure does not support multiple upload ids for an object. Hence we use object name itself as the uploadID.
```

Comment:
So, what happens when we try to upload multipart objects concurrently. Does NewMultipartUpload fail with upload already exists if an earlier request successfully created one?